src/images/neo4j.rs

```diff
@@ -113,16 +178,16 @@ fn neo4j(node: &Neo4jImage) -> Config<String> {
             format!("NEO4J_apoc_export_file_enabled=true"),
             format!("NEO4J_apoc_import_file_enabled=true"),
             format!("NEO4J_dbms_security_procedures_unrestricted=apoc.*,algo.*"),
-            format!("NEO4J_dbms_memory_heap_initial__size=64m"),
-            format!("NEO4J_dbms_memory_heap_max__size=512m"),
+            format!("NEO4J_server_memory_heap_initial__size=64m"),
+            format!("NEO4J_server_memory_heap_max__size=512m"),
             format!("NEO4J_apoc_uuid_enabled=true"),
-            format!("NEO4J_dbms_default__listen__address=0.0.0.0"),
+            format!("NEO4J_server_default__listen__address=0.0.0.0"),
             format!(
-                "NEO4J_dbms_connector_bolt_listen__address=0.0.0.0:{}",
+                "NEO4J_server_bolt_listen__address=0.0.0.0:{}",
                 &node.bolt_port
             ),
-            format!("NEO4J_dbms_allow__upgrade=true"),
-            format!("NEO4J_dbms_default__database=neo4j"),
+            format!("NEO4J_initial_dbms_default__database=neo4j"),
+            format!("NEO4J_dbms_security_auth__minimum__password__length=4"),
```

Comment:
we will need to check which version the neo4j binary is on and then decide which values to use